stories/Socials/Socials.stories.jsx

```diff
@@ -0,0 +1,45 @@
+import Socials from '../../components/Socials';
+
+const Template = args => <Socials {...args} />;
+
+const allItems = [
+  {
+    name: 'facebook',
+    url: 'https://facebook.com',
+    description: 'facebook description of the purpose of this icon',
+  },
+  {
+    name: 'twitter',
+    url: 'https://twitter.com',
+    description: 'twitter description of the purpose of this icon',
+  },
+  {
+    name: 'youtube',
+    url: 'https://youtube.com',
+    description: 'youtube description of the purpose of this icon',
+  },
+];
+
+export const AllItems = Template.bind({});
+AllItems.args = {
+  items: allItems,
+};
+
+export const AllItemsWithoutBox = Template.bind({});
+
+AllItemsWithoutBox.args = {
```

Comment:
I don't think this name would be good because the pattern already comes with all the items. the difference with this property is that it is not inside a box